docs/reference/docs/bulk.asciidoc

```diff
@@ -246,20 +246,20 @@ NOTE: Only the shards that receive the bulk request will be affected by
 `refresh`. Imagine a `_bulk?refresh=wait_for` request with three
 documents in it that happen to be routed to different shards in an index
 with five shards. The request will only wait for those three shards to
-refresh. The other two shards of that make up the index do not
+refresh. The other two shards that make up the index do not
 participate in the `_bulk` request at all.
 
 [float]
 [[bulk-update]]
 === Update
 
-When using `update` action `retry_on_conflict` can be used as field in
+When using `update`, the action `retry_on_conflict` can be used as a field in
```

Comment:
In this case we're altering the meaning: `retry_on_conflict` is a parameter / field that can be used to influence the behavior of the `update` action.

Reply:
Fixed